react-graphql-client/src/App.js

```diff
@@ -219,6 +220,50 @@ class App extends Component {
     });
   }
 
+  removeLineItemInCart(lineItemId) {
+    const checkoutId = this.state.checkout.id;
+
+    return this.props.client.send(gql(this.props.client)`
+      mutation ($checkoutId: ID!, $lineItemIds: [ID!]!) {
+        checkoutLineItemsRemove(checkoutId: $checkoutId, lineItemIds: $lineItemIds) {
+          userErrors {
+            message
+            field
+          }
+          checkout {
```

Comment:
Looks like there's the potential for this to be a fragment that's re-used across a couple queries?